### ISSUE TITLE
Dropdown caret is not vertically aligned for bootstrap 5

### ISSUE THREAD

**Issue by craigvantonder** [jgalindosl/vue-multiselect-bootstrap-theme#3]
The following declaration:

https://github.com/jgalindosl/vue-multiselect-bootstrap-theme/blob/51d36b6e1cf9681a61dcfce3b7947e863a3fe1ed/dist/vue-multiselect-bootstrap5.scss#L248

Is not consistent with the source value of 65%:

https://github.com/shentao/vue-multiselect/blob/47f36859b657693a06adaacaea0be67bd1385812/src/Multiselect.vue#L647

In fact, overriding the value does not seem to be necessary and although I have not tested this in bootstrap 3/4, simply removing it reverts the style to the source definition and results in the correct vertical alignment:

    .multiselect__select:before {
      position: relative;
      right: 0;
      /*top: 50%;*/
      color: $vue-multiselect-arrow-color;
      border-style: solid;
      border-width: $vue-multiselect-arrow-size $vue-multiselect-arrow-size 0 $vue-multiselect-arrow-size;
      border-color: $vue-multiselect-arrow-color transparent transparent transparent;
      content: "";
    }

Before (too high):

![Screenshot_2024-07-28_00-48-42](https://github.com/user-attachments/assets/21b449fb-7534-4123-961f-e85c4136ca0a)
![Screenshot_2024-07-28_00-54-06](https://github.com/user-attachments/assets/9a81664c-5047-4552-9eb6-709ad010876c)

After (vertically centered):

![Screenshot_2024-07-28_00-55-36](https://github.com/user-attachments/assets/0d30f341-669d-4fc8-ab98-bb99ed6d3440)
![Screenshot_2024-07-28_00-54-32](https://github.com/user-attachments/assets/3b12bc7e-c4ea-4cf5-9e3a-d27c0493d912)

EDIT: This is with `"bootstrap": "^5.3.3",`

**Comment by jgalindosl** [comment]
Thanks.  A new tag has been pushed with this fix.

**Comment by craigvantonder** [comment]
> Thanks. A new tag has been pushed with this fix.

I use vue and bootstrap a lot. The multiselect component is great solution but I have always wished that it accommodated for bootstraps styling, offering this repository as a drop in stylesheet was a great idea. Thank you for the time and effort that you have put into it thus far.

I had noticed a few discrepancies over and above what I mentioned above. That said, I am using your stylesheet as a base doing some specific overwrites so as to accommodate for the -sm size variation of Bootstrap select component. As a result, I did not much time looking at the current style sheet given my own use case. I am however happy to try to work on some of these at some point, if you ever need a hand or would like to see what changes were made for -sm then give me a shout.

**Comment by jgalindosl** [comment]
I would be more than willing to review and accept a pull request on your changes so we can help anyone else interested in them.  We could merge your changes with the current scss. No problem

**Comment by craigvantonder** [comment]
> I would be more than willing to review and accept a pull request on your changes so we can help anyone else interested in them. We could merge your changes with the current scss. No problem

I'll take a proper look and put something together when I have got the chance to focus on it.

The sm variation is a bit of a tricky one, most especially when it comes to the tagging feature, I did not find a reliable way to determine if tagging was being used by the component so as to apply specific changes for this use case when compared to the standard usage. I had worked around the lack of available differentiator by adding a data attribute into the component when tagging applies then relying on its presence to make the required changes. So it was not an ideal solution in my mind, but for me that does not matter and was fine.
